dhis-2/dhis-api/src/main/java/org/hisp/dhis/program/message/ProgramMessage.java

```diff
@@ -65,6 +67,10 @@ public class ProgramMessage extends BaseIdentifiableObject implements Serializab
   @JsonSerialize(as = BaseIdentifiableObject.class)
   private Enrollment enrollment;
 
+  @JsonIgnore private TrackerEvent trackerEvent;
```

Comment:
Some comments would be good as this will stay for a while AFAIK. `event` is now not used for persistence but only for the view, right? The other two fields are only used for persistence but not for the view, right? That's what I would document as that is not straightforward when looking at the class. A `@JsonProperty getEvent()` instead of an `event` field might also make this clearer.